.circleci/continue_config.yml

```diff
@@ -453,18 +458,64 @@ jobs:
       - run: ./gradlew --no-daemon integrationTest -x test   -Pspark.version=<< parameters.spark-version >>
       - run:
           when: on_fail
-          command: cat build/test-results/integrationTest/TEST-*.xml
+          command: cat app/build/test-results/integrationTest/TEST-*.xml
       - run: ./gradlew --no-daemon jacocoTestReport
       - store_test_results:
-          path: build/test-results/integrationTest
+          path: app/build/test-results/integrationTest
       - store_artifacts:
-          path: build/reports/tests/integrationTest
+          path: app/build/reports/tests/integrationTest
           destination: test-report
       - save_cache:
           key: v1-integration-spark-{{ .Branch }}-{{ .Revision }}
           paths:
             - ~/.gradle
-  
+
+  integration-test-databricks-integration-spark:
+    parameters:
+      spark-version:
+        type: string
+    working_directory: ~/openlineage/integration/spark
+    machine:
+      image: ubuntu-2004:current
+    resource_class: large
+    environment:
+      TESTCONTAINERS_RYUK_DISABLED: "true"
+      JDK8_HOME: /usr/lib/jvm/java-8-openjdk-amd64
+    steps:
+      - *checkout_project_root
+      - when:
+          condition:
+            equal: [ enabled, << pipeline.parameters.databricks-test >> ]
+          steps:
+            - restore_cache:
+                keys:
+                  - v1-integration-spark-{{ .Branch }}-{{ .Revision }}
+                  - v1-integration-spark-{{ .Branch }}
+            - attach_workspace:
+                at: ~/.m2/repository/io/openlineage/
+            - run: |
+                sudo update-alternatives --set java /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java
+                sudo update-alternatives --set javac /usr/lib/jvm/java-8-openjdk-amd64/bin/javac
+            - run: ./gradlew shadowJar -x test
+            - run: ./gradlew --no-daemon databricksIntegrationTest -x test   -Pspark.version=<< parameters.spark-version >> -PdatabricksHost=$DATABRICKS_HOST -PdatabricksToken=$DATABRICKS_TOKEN
+            - run:
+                when: on_fail
+                command: cat app/build/test-results/databricksIntegrationTest/TEST-*.xml
```

Comment:
Is this neccessary? There is `store_artifacts` step below that should contain the same and is more convenient to browse.

Reply:
It's been added within spark integration tests as well and I wanted to be consistent with that. 
Assuming this was failing for over a year and no-one has noticed that, I don't think it's necessary. at all